src/schema/sources.ts

```diff
@@ -409,6 +409,11 @@ export const typeDefs = /* GraphQL */ `
       Paginate first
       """
       first: Int
+
+      """
+      Source type (machine/squad)
+      """
+      type: String
```

Comment:
We can make this an enum right so it is validated properly? You can use `toGQLEnum` to reuse TS enum as well.